_docs-sources/faq/index.md

```diff
@@ -0,0 +1,66 @@
+import Card from "/src/components/Card"
```

Comment:
You might want to add front-matter with `hide_table_of_contents: true` here, which should result in a centered page layout.

Reply:
Done in 4de5e1d

Reply:
Done in 4de5e1d5f18a406f1e8c01d4b426765e415df8bd